### DISCUSSION TITLE
SFTP panel in tabby (right-click a tab) for MacOS

### DISCUSSION THREAD

**Discussion by junwu-cao** [Eugeny/tabby#7999] [Q&A] [answered]
I cannot find SFTP panel when I right-click a tab in tabby 1.0.189 for MacOS (Ventura 13.2.1). The SFTP panel exists in Tabby 1.0.189 for Windows.  Is SFTP panel a Windows' feature, not for MacOS? Thanks.

**Comment by Eugeny** [comment, marked as answer]
SFTP panel is only available when using Tabby's built-in SSH client (not when using `ssh user@host...` in the shell). You can open the panel from the toolbar, context menu or the command palette

**Reply by junwu-cao** [reply]
Thanks a lot.,

**Reply by lawweiliang** [reply]
@junwu-cao, try to close the discussion if it was answered. Thanks.

**Comment by junwu-cao** [comment]
Resolved.
